File: Second year/Design and Programming Environments/TicketManagerJava/Networking/src/main/java/ObjectProtocol/AddEventRequest.java
package ObjectProtocol;

import DTOPackage.EventDTO;

public class AddEventRequest implements Request {
    private EventDTO eventDTO;
    public AddEventRequest(EventDTO eventDTO){
        this.eventDTO=eventDTO;
    }
    public EventDTO getEventDTO(){
        return eventDTO;
    }
}

File: Second year/Design and Programming Environments/TicketManagerCSharp/Networking/Requests/AddEventRequest.cs
using Networking.DTOS;
using System;

namespace Networking.Requests
{
    [Serializable]
    public class AddEventRequest : Request
    {
        private EventDTO eventDTO;

        public AddEventRequest(EventDTO eventDTO)
        {
            this.eventDTO = eventDTO;
        }

        public EventDTO EventDTO { get => eventDTO; set => eventDTO = value; }
    }
}